_posts/2022-08-05-hacking-for-cruise-control-ui.md

```diff
@@ -0,0 +1,335 @@
+These custom resources provide great support for declaritive rebalancing.
+That being said, rather than working with custom resources, you might prefer to interact with applications like Cruise Control through a graphical user interface.
+LinkedIn provides such an interface with their [Cruise Control UI](https://github.com/linkedin/cruise-control-ui) application which supports several Cruise Control operations like viewing Kafka cluster status and executing partition rebalances, all with just a couple button clicks.
+Although Strimzi does not offer direct support for the [Cruise Control UI](https://github.com/linkedin/cruise-control-ui) it can still be run with and connected to a Strimzi-managed Cruise Control instance.
+
+This post will walk through how to do so! 
+
+**Warning**
+
+Setting up the [Cruise Control UI](https://github.com/linkedin/cruise-control-ui) as described in this blog post requires disabling some of the Cruise Control REST API security settings.
+Having these security settings disabled exposes the Cruise Control API to potentially destructive Cruise Control operations that are not coordinated or supported by Strimzi Operator.
+For example, it would allow users to manually decomission brokers and trigger partition rebalances, operations which could interfere with Strimzi operations. 
+If using the Cruise Control UI with a Strimzi-managed Kafka cluster, it is best to stick with using it solely for monitoring your cluster, not for operating on it.
+That being said, use at your own risk!
+
+![Cruise Control Frontend](/assets/images/posts/2019-08-08-cruise-control-frontend.png)
+
+
+## Motivation
+
+We have been getting a lot of requests for how to set up the Cruise Control UI with a Strimzi-managed Kafka cluster.
+Maybe you want a pretty dashboard to view your cluster status.
+Maybe you want to compare optimization proposals without having to open a text editor.
+Maybe you are just a visual person.
+Regardless of the reason, we want to provide some basic instructions for getting you started!
+
+## Prerequisites
+
+For starters we will need a Strimzi-managed Kafka cluster with Cruise Control enabled.
+
+![](/assets/images/posts/2022-10-31-hacking-for-cruise-control-ui-1.png)
+
+Since Strimzi does not support creating custom Cruise Control API user roles we will need to disable HTTP basic authentication settings.
+
+We can do so by configuring Cruise Control within a `Kafka` resource like this:
+
+``` 
+# kafka-cruise-control.yaml
+apiVersion: kafka.strimzi.io/v1beta2
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  cruiseControl:
+    config:
+      webserver.security.enable: false
+```
+
+After configuring our Strimzi-managed Kafka cluster, we can deploy and connect the [Cruise Control UI](https://github.com/linkedin/cruise-control-ui) in one of two ways:
+
+* METHOD ONE: Inside a dedicated Cruise Control pod
+* METHOD TWO: Outside a dedicated Cruise Control pod
```

Comment:
I know we've iterating over these names a few times already and I wonder if `Customizing the Cruise Control pod` and `Creating a dedicated Cruise Control UI pod` would be even clearer. WDYT?